src/assets/metadata/aws.json

```diff
@@ -194,43 +894,166 @@
+        },
+        {
+          "name": "enable_dns64",
+          "type": "Boolean",
+          "displayName": "Enable DNS64",
+          "description": "Whether DNS queries made to the Amazon-provided DNS Resolver<br>in this subnet should return synthetic IPv6 addresses for IPv4-only destinations.",
+          "url": "https://docs.aws.amazon.com/vpc/latest/userguide/nat-gateway-nat64-dns64.html"
+        },
+        {
+          "name": "assign_ipv6_address_on_creation",
+          "type": "Boolean",
+          "displayName": "Assign IPv6 address on creation",
+          "description": "Specify true to indicate that network interfaces created in the specified subnet should be assigned an IPv6 address.",
+          "url": "https://docs.aws.amazon.com/vpc/latest/userguide/configure-subnets.html"
+        },
+        {
+          "name": "ipv6_cidr_block",
+          "type": "String",
+          "displayName": "IPv6 CIDR block",
+          "description": "The IPv6 network range for the subnet, in CIDR notation",
+          "url": "https://docs.aws.amazon.com/vpc/latest/userguide/configure-subnets.html",
+          "rules": {
+            "regex": "[a-fA-F0-9]{4}(:[a-fA-F0-9]{4}){7}"
+          }
+        },
+        {
+          "name": "ipv6_native",
+          "type": "Boolean",
+          "displayName": "IPv6 native",
+          "description": "Whether to create an IPv6-only subnet.",
+          "url": "https://docs.aws.amazon.com/vpc/latest/userguide/configure-subnets.html"
+        },
+        {
+          "name": "enable_resource_name_dns_a_record_on_launch",
+          "type": "Boolean",
+          "displayName": "Enable ressource name DNS A record on launch",
+          "description": "Whether to respond to DNS queries for instance hostnames with DNS A records.",
+          "url": "https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/rrsets-working-with.html"
+        },
+        {
+          "name": "enable_resource_name_dns_aaaa_record_on_launch",
+          "type": "Boolean",
+          "displayName": "Enable ressource name DNS AAAA record on launch",
+          "description": "Whether to respond to DNS queries for instance hostnames with DNS AAAA records.",
+          "url": "https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/rrsets-working-with.html"
+        },
+        {
+          "name": "private_dns_hostname_type_on_launch",
+          "type": "String",
+          "displayName": "Private DNS hostname type on launch",
+          "description": "The type of hostnames to assign to instances in the subnet at launch.",
+          "url": "https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-naming.html"
+        },
+        {
+          "name": "gateway_id",
+          "type": "Link",
+          "linkType": "Default",
+          "linkRef": "aws_internet_gateway",
+          "displayName": "Gateway ID",
+          "url": "https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Internet_Gateway.html"
```

Comment:
According to the [terraform documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/subnet), there is no `gateway_id` parameter. It was added so that the subnets could be linked to the gateway in the modelizer. However, it will cause incorrect terraform code to be generated so I deleted the paramater.